capabilities/communication/send-email/maps/postmark.suma

```diff
@@ -53,52 +53,3 @@ map SendEmail {
   }
 }
 
-map SendTemplatedEmail {
-  http POST "/email/withTemplate" {
-    security "server_token"
-
-    request "application/json" {
-      body {
-        From = input.from
-        To = input.to
-        Subject = input.subject
```

Comment:
In new separated map, you're not using `Subject`, is this on purpose?

Reply:
Yes. It doesn't need to be there, because subject can be set directly in template in Postmark app.
Honestly I have no idea why I hard there O:)

<img width="1415" alt="Screenshot 2021-09-14 at 21 18 38" src="https://user-images.githubusercontent.com/959390/133320443-e3ee0004-4d55-485a-a308-22d631795466.png">